src/components/external-link/external-link.tsx

```diff
@@ -9,7 +9,13 @@ interface Props {
 }
 
 const ExternalLink = (props: Props) => {
-  const { className, href, children } = props;
+  const { className, children } = props;
+
+  let href = props.href.trim();
+  const urlRegex = /^https:\/\//i; // Starts with https:// (case insensitive)
```

Comment:
Mhh, we already do this precaution in https://github.com/api3dao/api3-dao-dashboard/blob/main/src/pages/proposal-commons/proposal-details/proposal-details.tsx#L39 but we also accept `http`

Any reason why only accept https?

Reply:
Just for some more context, this PR is more of a precautionary change (I created a proposal with a dodgy javascript URL and saw that it did not create a link for it). 

W.r.t. `http`, ah I missed the fact that we already allow them, so I will make the change to bring them back.

Reply:
Also, it would be nice to show a warning (at least to console). WDYT?

Reply:
Sounds good 👍